target_chains/starknet/contracts/src/pyth.cairo

```diff
@@ -300,6 +310,54 @@ mod pyth {
                 )
         }
 
+        fn wormhole_address(self: @ContractState) -> ContractAddress {
+            self.wormhole_address.read()
+        }
+
+        fn fee_contract_address(self: @ContractState) -> ContractAddress {
+            self.fee_contract_address.read()
+        }
```

Comment:
hmmm maybe `fee_token_address`? I was puzzled a bit when i read it.

Reply:
Renamed.